docs/ref/contrib/admin/filters.txt

```diff
@@ -0,0 +1,162 @@
+=================
+Admin List Filter
+=================
+
+.. currentmodule:: django.contrib.admin
+
+.. attribute:: ModelAdmin.list_filter :noindex:
```

Comment:
Added `:no-index:` here since Sphinx was showing a warning for two attributes with same names. 
Should it be attribute though in this file?